src/GQLQueries/recentSubmit.ts

```diff
@@ -1,5 +1,5 @@
 const query = `#graphql
-query ($username: String!, $limit: Int) {
+query getRecentSubmissions($username: String!, $limit: Int) {
```

Comment:
I gave these queries names so that I could reference them in the handlers.ts file, I don't see that it broke anything

Reply:
Yeah! It works absolutely fine😊